applications/HDF5Application/custom_utilities/factor_elements_and_conditions_utility.h

```diff
@@ -0,0 +1,149 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 license: HDF5Application/license.txt
+//
+//  Main author:    Michael Andre, https://github.com/msandre
+//
+
+#if !defined(KRATOS_FACTOR_ELEMENTS_AND_CONDITIONS_UTILITY_H_INCLUDED)
+#define KRATOS_FACTOR_ELEMENTS_AND_CONDITIONS_UTILITY_H_INCLUDED
+
+// System includes
+#include <vector>
+#include <string>
+#include <utility>
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/element.h"
+#include "includes/condition.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+///@name Kratos Classes
+///@{
+
```

Comment:
maybe some doxygen of what this utility does?

Reply:
added